metrics/server.go

```diff
@@ -724,10 +732,10 @@ func getGRPCDashboard(w http.ResponseWriter, r *http.Request) {
 	_, _ = w.Write(dashboardBytes)
 }
 
-// putExperimentResult handles PUT /experimentResult with query parameter application=namespace/name
+// putExperimentResult handles PUT /testResult with query parameter test=name and namespace=namespace
```

Comment:
Note gap in function name `putExperimentResult` and endpoint `/testResult with query parameter test=name and namespace=namespace`